src/components/alert/Alert.tsx

```diff
@@ -25,11 +24,9 @@ interface AlertProps extends DialogBaseProps {
   type?: Type;
 }
 
-const Alert = ({ primaryAction, secondaryAction, title, body, type = 'confirm', ...otherProps }: AlertProps) => {
-  const restProps = omit(otherProps, ['primaryAction', 'secondaryAction']);
-
+const Alert = ({ primaryAction, secondaryAction, title, body, type = 'confirm' }: AlertProps) => {
   return (
-    <DialogBase {...restProps} scrollable={false} size="small">
```

Comment:
Don't you still need {...otherProps} here? 😅

Reply:
🙈 Woopsie